app/controllers/maps_controller.rb

```diff
@@ -8,6 +8,7 @@ def create
     campaign = Campaign.find(params[:campaign_id])
     map = campaign.maps.new(map_params)
     if map.save
+      map.center_image
```

Comment:
I could see this being set `before_create` on `Map`, but if you'd prefer to avoid callbacks could you also add this call to the `dev:prime` task

Reply:
Unfortunately, a `before_save` callback doesn't work for this because in Rails 6 ActiveStorage has not attached the image yet. 

I opted not to do an `after_save` because image analysis happens in a background job and I saw some inconsistent behavior.

Reply:
I've added it to the `dev:prime` task in f47ec48a8d725dd187ca0f84c2295f4437a6d591